server/etcdserver/server.go

```diff
@@ -904,10 +904,26 @@ func (s *EtcdServer) revokeExpiredLeases(leases []*lease.Lease) {
 	})
 }
 
+// isActive checks if the etcd instance is still actively processing the
+// heartbeat message (ticks). It returns false if no heartbeat has been
+// received within 3 * tickMs.
```

Comment:
Why 3 election ticks? I think the correct value is somewhere between `TickMs` (default 100ms) and `ElectionMs` (default 1s), meaning the 3 makes sense for default config, but I would argue it can be problematic for cases where `ElectionMS` < 3* `TicksMs`

Reply:
> Why 3 election ticks?

Read https://github.com/etcd-io/etcd/pull/18428#discussion_r1712624482.

> I would argue it can be problematic for cases where `ElectionMS` < 3* `TicksMs`

I am not worry about this.
- checking activity should be only related to `tickMs`.
- also if `ElectionMS` < 3* `TicksMs`, it means it's a very inappropriate election value.

Reply:
> also if ElectionMS < 3* TicksMs, it means it's a very inappropriate election value.

Then let's codify that is is inappropriate. Either working or failing validation.